docs/user/README.md

```diff
@@ -18,6 +18,14 @@ By default, the cluster will be given the name `kind-1`. "1," here, is the
 default context name. 
 Use the `--name` flag to assign the cluster a different context name.
 
+**Note**: If you are running `kind` of MacOS or Windows then it is recommended
+you have at least 4GB of RAM and disk space dedicated to the virtual machine 
```

Comment:
what are we basing the estimate here - 4GB?
is this the minimum for a single node setup?

also how much disk space do we require?

Reply:
4 GB of ram seemed to work to resolve #165 and that's what the Docker installation instructions for macos mention.
I did add that the estimate assumes a single-node cluster.

Reply:
4GB RAM and another 4GB of disk space?
isn't 4GB of disk too low?